css/src/components/pagination.css

```diff
@@ -28,8 +28,7 @@
 .Pagination-MetricInput {
   width: var(--spacing-4);
   margin-right: var(--spacing);
-  padding-left: var(--spacing-m);
-  padding-right: var(--spacing-m);
+  padding: 6px var(--spacing);
```

Comment:
Please only give `8px` padding to left/right. Top/Bottom padding is not required. it will be handled automatically by giving height.